projects/storefrontapp-e2e-cypress/cypress/integration/regression/cart.e2e-spec.ts

```diff
@@ -1,5 +1,6 @@
 import * as cart from '../../helpers/cart';
 import * as alerts from '../../helpers/global-message';
+import { apiUrl, login } from '../../support/utils/login';
```

Comment:
### For authors

Respond to every comment. Don't merge PR with open comments without any response.